base_custom_attribute/views/attribute_attribute_view.xml

```diff
@@ -1,36 +1,60 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data>
 
         <record id="attribute_attribute_form_view" model="ir.ui.view">
             <field name="name">attribute.attribute.form</field>
             <field name="model">attribute.attribute</field>
             <field name="arch" type="xml">
                 <form string="Attribute">
-                    <group>
+                    <group col="2" class="col-6">
                         <field name="field_description"/>
-                        <field name="name" attrs="{'readonly':[('create_date', '!=', False)]}"/>
-                        <field name="attribute_type" />
-                        <field name="model_id" />
-                        <field name="serialized" attrs="{'invisible':[('attribute_type', '=', 'multiselect')]}"/>
-                        <field name="size" attrs="{'invisible':[('attribute_type', '!=', 'char')]}"/>
+                        <field name="name" attrs="{'readonly': [('create_date', '!=', False)]}"/>
+                        <field name="attribute_type"/><newline/>
+                        <field name="attribute_group_id" options="{'no_create': 1}" />
+                        <field name="sequence"/>
+                        <field name="attribute_set_ids"
+                            widget="many2many_tags"
+                            domain="[('model_id', '=', context.get('default_model_id'))]"
+                            options="{'no_create': 1}"
+                            required="not context.get('from_attribute_set')"
```

Comment:
@hpar @clementmbr I do not know if it's a good idea to make this field required?
Indeed maybe you want to create your attribute (all of them and after create your attribute set)?

Reply:
I don't really manage to imagine this use case to create first the attributes and then the attribute set... But if we what to cover this kind of scenario, we might add some "Action button" on attributes tree view to give an attribute set to many attributes at once, might we?

In my opinion, I think I like this field to be required as it will help new users to understand what are the necessary fields to use an attribute (as it is impossible to use an attribute without an attribute set, a new user can create an attribute without attribute set at first and then loosing some time to understand why his attribute doesn't appear anywhere...)

Reply:
> I don't really manage to imagine this use case to create first the attributes and then the attribute set.

For me it make sense to first create the attributes in a row, then figure out how to group them.